en/firefox/features/shared.ftl

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/
+
+features-shared-a-better-internet-experience = A better internet experience
+
+features-shared-browse-faster = Browse faster
+features-shared-your-favorite-extensions = Your favorite extensions
```

Comment:
https://github.com/mozilla-l10n/www-l10n/blob/cbbf2a7f5e943b060dc58d0fa88a32d46352a8e6/en/firefox/features/index.ftl#L25 - the index file (as well as staging) says `Your favorite add-ons and extensions`.

Reply:
That's the old string, they've changed it to just "Your favorite extensions"